parcel_test.go

```diff
@@ -31,11 +32,36 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db") // настройте подключение к БД
+	if err != nil {
+		require.NoError(t, err)
+	}
+	defer db.Close()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
-	// add
+	id, err := store.Add(parcel)
+
+	require.NoError(t, err)
+	require.NotNil(t, id)
+
+	parcels, err := store.Get(id)
+
+	require.NoError(t, err)
+	assert.Equal(t, parcel.Client, parcels.Client)
+	assert.Equal(t, parcel.Status, parcels.Status)
+	assert.Equal(t, parcel.Address, parcels.Address)
+	assert.Equal(t, parcel.CreatedAt, parcels.CreatedAt)
+
+	err = store.Delete(id)
+	if err != nil {
+		require.NoError(t, err)
+	}
+
+	_, err = store.Get(id)
+
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```